src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -120,6 +120,27 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                             IsAzureDB = isAzureDB
                         };
 
+                        // Collect the Database properties information
+                        if (!requestParams.IsNewObject)
+                        {
+                            var smoDatabase = dataContainer.SqlDialogSubject as Database;
+                            databaseViewInfo.ObjectInfo = new DatabaseInfo()
+                            {
+                                Name = smoDatabase.Name,
+                                CollationName = smoDatabase.Collation,
+                                DateCreated = smoDatabase.CreateDate.ToString(),
+                                LastDatabaseBackup = smoDatabase.LastBackupDate == DateTime.MinValue ? "None" : smoDatabase.LastBackupDate.ToString(),
```

Comment:
I've been using nulls or Empty to represent something not being supported at all, so it would be left out of the UI. Like leaving out the Owner field in Azure dialogs.

Also, localizing None in the service side let's us more easily check it during Update operations later. So if you send a set of properties with None to the frontend and get the same None value back, then you skip updating that value.

Reply:
This is actually a case where null vs undefined would be useful...

Can you verify that STS is picking up the correct display language? We have so few features in STS that give UI strings that I'm not actually sure, and I know loc is a big gap for a lot of our stuff.

In the end I still don't think that STS should be doing loc stuff in general since it just makes the code here more complex (let the UI be concerned about UI stuff) - but if we can't find a simple solution for handling this then I'm ok with this as long as STS is at least handling the language stuff correctly.

Reply:
We talked offline about it, and it makes more sense to check if a property is present in the DatabaseViewInfo object, rather than checking if it's undefined or empty, to determine if it should be rendered in the frontend. Charles will take a look at our RPC serializer to see if properties don't get defined if we receive a null value.

Reply:
Chatted offline with Cory about this - you can leave this as it is for now. I need to investigate whether we can use another pattern for detecting "unsupported" options vs. empty/default options.

Reply:
Hmm, I didn't think this way, as current experience of general tab would be a read-only! I'll take a look at your asks! Thanks :)